src/pages/search/project_search/ProjectStatusFilter.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, MenuItem, TextField, Typography } from '@mui/material';
+import { ProjectSearchParameter } from '../../../api/cristinApi';
```

Comment:
Gir deg samme tips som Anette fikk på sin PR:

anbefaler å sette opp vscode (om det er det du bruker?) til å automatisk sortere imports: https://dev.to/shanesc/how-to-sort-and-cleanup-imports-on-save-in-vs-code-52p1
Hjelper både med å sortere, og å fjerne ubrukte imports